scout/server/extensions/ldap_extension.py

```diff
@@ -1,72 +1,65 @@
 import logging
-import ssl
 
-from flask_ldapconn import LDAPConn
-from ldap3 import ALL, SYNC, Server, Tls
+from flask_ldap3_login import LDAP3LoginManager
 
 LOG = logging.getLogger(__name__)
 
 
-class LdapManager(LDAPConn):
-    """Interface to LDAP login"""
+class LdapManager:
+    """Interface to LDAP login using flask-ldap3-login"""
+
+    def __init__(self):
+        self.ldap_manager = LDAP3LoginManager()
+        self.app = None
 
     def init_app(self, app):
-        ssl_defaults = ssl.get_default_verify_paths()
+        """Initialize LDAP manager with Flask app config."""
+        self.app = app
 
-        # Default config
-        app.config.setdefault("LDAP_SERVER", "localhost")
+        app.config.setdefault("LDAP_HOST", "localhost")
         app.config.setdefault("LDAP_PORT", 389)
-        app.config.setdefault("LDAP_BINDDN", None)
```

Comment:
Compared old keys with new ones. Thank goodness they are named the same across libraries:

<img width="863" height="779" alt="image" src="https://github.com/user-attachments/assets/f87755f6-55a4-4317-9893-7be387f89f74" />